.github/workflows/weekly-version-bump.yml

```diff
@@ -0,0 +1,73 @@
+name: Bump versions of all DJ Services and Clients.
+on:
+  schedule:
+    - cron: "0 0 * * MON"
+  workflow_dispatch:
+    inputs:
+      bump:
+        type: choice
+        description: "Hatch version option (default: alpha)"
+        required: true
+        default: alpha
+        options:
+          - major
+          - minor
+          - patch
+          - alpha
+          - beta
+
+jobs:
+  publish:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.10']
+    runs-on: 'ubuntu-latest'
+    defaults:
+      run:
+        working-directory: .
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Hatch
+        run: |
+          python -m pip install --upgrade pip
+          pip install hatch
+
+      - name: Configure Git
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+
+      - name: Bump versions and make a PR
+        run: |
+          echo "Bump DJ Server version ..."
+          cd datajunction-server
+          hatch version ${{ github.event.inputs.bump }}
+          export NEW_VERSION=v$(hatch version)
+
+          echo "Copy version to DJ Query Service ..."
+          cp ./datajunction_server/__about__.py ../datajunction-query/djqs/
+
+          echo "Apply to DJ Python Client ..."
+          cp ./datajunction_server/__about__.py ../datajunction-clients/python/datajunction/
```

Comment:
Hmm, wait, these might not have the same versions right? The client and the server are already not in sync in terms of released versions, and plus in the future if someone does a manual release of either the client or the server alone, these will get out of sync.

Reply:
I'd like to put them in sync. But you are right that someone could always run one of these locally and ship to pypi... hmmm. I can switch to making these versions independent.